7.js

```diff
@@ -0,0 +1,61 @@
+const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+'Jul', 'Aug', 'Sep', 'Oct', 'Nov',
+'Dec']
+
+function formatDate (date) {
```

Comment:
I note that you're using an ES6 const declaration above, but an ES5 function declaration.  Consider enforcing internal version consistency in your modules (preferably in your project).